stl/CMakeLists.txt

```diff
@@ -249,6 +249,7 @@ endforeach()
 # Objs that exist in both libcpmt[d][01].lib and msvcprt[d].lib.
 set(IMPLIB_SOURCES
     ${CMAKE_CURRENT_LIST_DIR}/src/filesystem.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/src/format.cpp

```

Comment:
`stl\msbuild\stl_base\stl.files.settings.targets` also needs to be edited (in addition to MSVC-internal files).